programs/ziti-edge-tunnel/netif_driver/linux/tun.c

```diff
@@ -145,21 +201,34 @@ static void find_dns_updater() {
     };
 
     static struct dns_cmd dns_cmds[] = {
-            {
-                    .path = "/usr/bin/resolvectl",
-                    .update_fn = dns_update_resolvectl,
-            },
-            {
-                    .path = "/usr/bin/systemd-resolve",
-                    .update_fn = dns_update_systemd_resolve,
-            },
             {
                     .path = "/usr/sbin/resolvconf",
                     .update_fn = dns_update_resolvconf
             },
             {0}
     };
 
+    _cleanup_(sd_bus_flush_close_unrefp) sd_bus *bus = NULL;
```

Comment:
maybe refactor this check into a function and update `dns_cmd` to have a function to check